src/components/Header/header.module.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+  align-items: center;
+  background-color: #f0f0f0;
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem;
+  padding: 1rem;
+
+  .restro__details {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    .restro__title {
+      color: #000;
+      font-family: Open Sans;
+      font-size: 1.25rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: normal;
+      text-align: center;
+    }
+    .table__num {
+      color: #000;
+      font-family: Open Sans;

```

Comment:
define font family in the constant.scss and use it here

Reply:
Addressed